components/datadog/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@pipedream/datadog",
+  "version": "0.0.1",
+  "description": "Pipedream Datadog Components",
+  "main": "datadog.app.mjs",
+  "keywords": [
+    "pipedream",
+    "datadog"
+  ],
+  "homepage": "https://pipedream.com/apps/datadog",
+  "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+  "license": "MIT",
+  "dependencies": {
+    "@datadog/datadog-api-client": "^1.0.0-beta.9",
+    "lodash.pick": "^4.4.0",
+    "lodash.pickby": "^4.6.0"
```

Comment:
Should this be "lodash" instead of "lodash.pick" and "lodash.pickby"?

Reply:
Addressed